sale_product_multi_add/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "category": "Sale Management",
     "version": "15.0.1.0.0",
```

Comment:
Should we also bump the version number to `15.0.1.0.1`?

Reply:
@n17  I think yes we should update the version number to `15.0.1.0.1`

Reply:
@st-anagpal @n17 There is no need of updating module version as it will be done through ocabot.

The only case where you need it is when you have migration scripts

Reply:
@rousseldenis Ok noted.